pkg/controller/master/supportbundle/controller.go

```diff
@@ -19,6 +20,8 @@ import (
 	"github.com/harvester/harvester/pkg/settings"
 )
 
+const supportBundleExistTimeLimit = 10 * time.Minute
```

Comment:
Can we make the value longer to 30 minutes and configurable? We can create a setting for this.

Reply:
surely, sounds good!